tests/test_yaml_checker.py

```diff
@@ -147,23 +147,22 @@ def test_check_side_out_matches_ref_arg(
     assert check_side_out_matches_ref_arg(conf) == expected
 
 
-@pytest.mark.parametrize(
-    "yaml_file, expected",
-    [
-        ("pipeline_cpu1.yaml", True),
-        ("pipeline_cpu2.yaml", True),
-        ("pipeline_gpu1.yaml", True),
-        ("testing/sweep_manual.yaml", True),
-    ],
-    ids=[
-        "cpu1_pipeline",
-        "cpu2_pipeline",
-        "gpu1_pipeline",
-        "sweep_manual",
-    ],
-)
-def test_validate_yaml_config(
-    sample_pipelines: str, yaml_file: str, standard_data: str, expected: bool
-):
-    yaml_file = sample_pipelines + yaml_file
-    assert validate_yaml_config(yaml_file, standard_data) == expected
+# Commentting out this test for now as we auto-generate pipelines during the docs build
+# @pytest.mark.parametrize(
+#     "yaml_file, expected",
+#     [
+#         ("cpu_pipeline1.yaml", True),
+#         ("gpu_pipeline1.yaml", True),
+#         ("testing/sweep_manual.yaml", True),
+#     ],
+#     ids=[
+#         "cpu1_pipeline",
+#         "gpu1_pipeline",
+#         "sweep_manual",
+#     ],
+# )
+# def test_validate_yaml_config(
+#     sample_pipelines: str, yaml_file: str, standard_data: str, expected: bool
+# ):
+#     yaml_file = sample_pipelines + yaml_file
+#     assert validate_yaml_config(yaml_file, standard_data) == expected
```

Comment:
The code comment could do with some extra information, it doesn't explain under what circumstances the test can be reintroduced, nor does it say how auto-generation of pipelines relates to this test being commented-out. I could hazard a guess that it's something to do with sweep-pipelines not being auto-generated or something, but I'd rather not have to guess if someone already knows the answer.

If the test doesn't make sense to be reintroduced after the changes that have been done in the PR, it may be better to just remove the test entirely.

Reply:
I actually re-enabled it, but it will run with `pytest.mark.small_data` marking as it will require the generated full pipelines. It passes for me.